### ISSUE TITLE
[Feature Request] Add Retroarch's Remote Retropad Core.

### ISSUE THREAD

**Issue by dxdatabase** [cizia64/CrossMix-OS#128]
Can't find Remote Retropad Core in Retroarch. It allows you to launch a Retroarch game on your computer using the Trimui as a controller. Very nice for multiplayer games. 

https://docs.libretro.com/library/remote_retropad/

**Comment by dxdatabase** [comment]
And we can imagine a more universal solution to use the TrimUI as a gamepad, something like the homebrew [3DS Controller](https://github.com/CTurt/3DSController/tree/master) for 3DS, or [DS2key](https://github.com/sypherce/ds2key) for DS.

**Comment by cobaltgit** [comment]
The trimui wouldn't be a great controller because of the lack of L3/R3 and lack of true analogue triggers, I like your thinking though :)

**Comment by dxdatabase** [comment]
There is no HDMI output on TrimUI, so use it as a gamepad will allow to play retro games on a PC screen :)

**Comment by cizia64** [comment]
Hi, 

To have Remote Retropad in RA it would require to re-compile Retroarch.... And you'll be probably disapointed by the input lag induced byt the wifi.
Note that you can plug an USB gamepad on the TSP, you'll have better results than wifi emulated gamepad to play with a friend on the same TSP screen.

**Comment by dxdatabase** [comment]
Close friends only to play on the same TSP 🤣 

Just tried 3DS Controller on my 3DS XL, work nice, the lag is completely minimal, totally playable.

**Comment by Nevrdid** [comment]
Didn't try but this should allow use to set this :
https://www.virtualhere.com/usb_server_software

**Comment by toniroller** [comment]
Yo can do so using Moonlight, when paired and streaming, you are controlling the pc game with the Trimui.

**Comment by dxdatabase** [comment]
Yes, but a lot of computer CPU/GPU power is used for encoding & streaming to the TrimUI
